docs/DEVELOPING.md

```diff
@@ -12,6 +12,138 @@
 - [PostgreSQL](https://www.elastic.co/guide/en/enterprise-search/master/connectors-postgresql.html)
 - [S3](https://www.elastic.co/guide/en/enterprise-search/master/connectors-s3.html)
 
+## Confluence Connector
+
+The [Elastic Confluence connector](../connectors/sources/confluence.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+### Availability and prerequisites
+
+This connector is available as a **connector client** from the **Python connectors framework**. To use this connector, satisfy all [connector client requirements](https://www.elastic.co/guide/en/enterprise-search/master/build-connector.html).
+
+This connector is in **beta** and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.
+
+### Usage
+
+To use this connector as a **connector client**, use the **build a connector** workflow. See [Connector clients and frameworks](https://www.elastic.co/guide/en/enterprise-search/master/build-connector.html).
+
+For additional operations, see [Usage](https://www.elastic.co/guide/en/enterprise-search/master/connectors-usage.html).
+
+### Compatibility
+
+Confluence versions 7 or later are compatible with Elastic connector frameworks.
+
+### Setup and basic usage
+
+Complete the following steps to deploy the connector:
+
+1. [Gather Confluence details](#gather-confluence-details)
+2. [Configure Confluence connector](#configure-confluence-connector)
+
+#### Gather Confluence details
+
+Collect the information that is required to connect to your Confluence instance:
+
+- The domain where Confluence is hosted.
+- Username for the Confluence server or account email for the Confluence cloud.
+- Password for the Confluence server or API Token for the Confluence cloud.
+- SSL certificate for a secure connection
+
+#### Configure Confluence connector
+
+The following configuration fields need to be provided for setting up the connector:
+
+##### `data_source`
+
+Dropdown to determine the Confluence platform type. `Confluence Cloud` if Confluence cloud and `Confluence Server` if Confluence server. Default value is `Confluence Cloud`.
```

Comment:
Do we support Confluence Data Center in addition to Confluence Server? If so, let's change all our "Confluence Server" references to "Confluence Server/Data Center".

Reply:
No, the connector supports Confluence Cloud and Confluence Server only.